nixos/tests/systemd-boot.nix

```diff
@@ -39,6 +39,29 @@ in
     '';
   };
 
+  # Check that specialisations create corresponding boot entries.
```

Comment:
For testing something like this, where we don't actually boot the systemd entry, I'd prefer a python-only test suite that (for example) outputs systemd boot entries to a temporary boot directory.  But, our python script isn't currently set up to be easily testable in this way, so I think this suffices for the time being.

Reply:
Ack.